### ISSUE TITLE
Bootstrap Nexus during jenkins agent ci build tests

### ISSUE THREAD

**Issue by clemensutschig** [opendevstack/ods-quickstarters#449]
**Is your feature request related to a problem? Please describe.**
Although we build some jenkins agents during pr verfication - we should start nexus underneath and build the agents against it... this way we van verify that the agents do work with nexus.

**Comment by clemensutschig** [comment]
the more I think about this - the more painfull this is ... :)

1. we need nexus /configured (from https://github.com/opendevstack/ods-core/nexus) 
1. nexus image? ... or rebuild?
1. we need to bootstrap nexus before the agent builds
1. we need to stop this after the agent builds

**Comment by michaelsauter** [comment]
@clemensutschig I think GitHub Actions actually supports (2) by using `docker-compose.yml` files. Should work. For (1) - this will add extra time to the build but in theory it should work. I'm not that worried about (3) - pretty sure GitHub kills everything automatically after each run. So yes this will be work but it sounds doable and like a fun exercise :)

**Comment by clemensutschig** [comment]
> @clemensutschig I think GitHub Actions actually supports (2) by using `docker-compose.yml` files. Should work. For (1) - this will add extra time to the build but in theory it should work. I'm not that worried about (3) - pretty sure GitHub kills everything automatically after each run. So yes this will be work but it sounds doable and like a fun exercise :)

agreed, I think we should use the configure.sh magic - and the 5 mins all this takes is well invested, to really prove all agents do work ... :)
